spark-template-thymeleaf/src/main/java/spark/template/thymeleaf/ThymeleafTemplateEngine.java

```diff
@@ -76,6 +89,26 @@ public String render(ModelAndView modelAndView) {
         } else {
             throw new IllegalArgumentException("modelAndView.getModel() must return a java.util.Map");
         }
+    }
 
+    private static TemplateResolver createDefaultTemplateResolver(String prefix, String suffix) {
+        TemplateResolver defaultTemplateResolver = new TemplateResolver();
+        defaultTemplateResolver.setTemplateMode(DEFAULT_TEMPLATE_MODE);
+
+        if (prefix != null) {
```

Comment:
How do you feel about: 
defaultTemplateResolver.setPrefix(
        prefix != null ? prefix : DEFAULT_PREFIX
);
        
defaultTemplateResolver.setSuffix(
        suffix != null ? suffix : DEFAULT_SUFFIX
);